### PR TITLE
add Sign your work section

### PR DESCRIPTION
**Pull request by nicolas2bert**
Adding **Sign your work** section

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -247,6 +247,61 @@ The tags used in the commit message shall follow the same scheme as the tags
 present in the `WIP` branch names, described in the
 [Branching Scheme](#branching-guidelines).
 
+### Sign your work
+
+Signing your work involves adding a line at the end of your patch's git commit
```
Comment (LaurenSpiegel) [inline CONTRIBUTING.md]: I would simply say:
In order to contribute to the project, you must sign your work.  By signing your work, you certify to the statements set out in the Developer Certificate of Origin below (from [developercertificate.org](http://developercertificate.org/).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -247,6 +247,61 @@ The tags used in the commit message shall follow the same scheme as the tags
...
+    person who certified (a), (b) or (c) and I have not modified
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
+Then to sign your work, add this line to each of your git commit messages using
```
Comment (LaurenSpiegel) [inline CONTRIBUTING.md]: I would say:

Signing your work is easy.  Just add the following line at the end of each of your commit messages.  You must use your real name in your sign-off.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -247,6 +247,61 @@ The tags used in the commit message shall follow the same scheme as the tags
...
+your real name:
+
+```
+Signed-off-by: Jane Doe <jane.doe@email.com>
+```
+Alternatively, if `user.name` and `user.email` is set in git configs, you can
```
Comment (LaurenSpiegel) [inline CONTRIBUTING.md]: If your `user.name` and `user.email` are set in our git configs, you can sign each commit automatically by using the `git commit -s` command.

**Comment by DavidPineauScality** [comment]
Given that this PR has received enough reviews, here is a question relative to the subject: Did you think about GPG signing of commits ? (I'm personally not a huge fan, but it is stricter than the simple git signed-off-by)
I'll merge once I get the answer

**Comment by NicolasT** [comment]
@DavidPineauScality My .02: that adds quite a burden to contributors, and doesn't really add much unless the other party's key is somehow part of your (personal) 'web of trust'.

**Comment by DavidPineauScality** [comment]
sure, hence "not a fan of" :)

**Pull request merged**